app/admin/book.rb

```diff
@@ -4,11 +4,11 @@
     column :title
     column :author
     column :published_date
-    column :price
+    column :price_cents
```

Comment:
You may want to change this. It's more user-friendly to show this column in dollars.